conf/data/configs/application.ini

```diff
@@ -79,8 +81,10 @@ admin_password = "12345"
 allow_admin_panel = On
 allow_two_factor_auth = Off
 force_two_factor_auth = Off
-allow_universal_login = Off
+hide_x_mailer_header = Off
 admin_panel_host = ""
+admin_panel_key = "admin"
```

Comment:
Uuuh, is this like the password to access some sort of admin interface o.O ?

Reply:
As I understand, it is a layer of protection to protect the admin panel.
By default, the administration panel is accessible with this address: `http://DOMAIN.TLD/rainloop/app/?admin`
if you set `admin_panel_key = "admin"` to `admin_panel_key = "my-secret-admin"`, the admin panel will be accessible with the folowing address:  `http://DOMAIN.TLD/rainloop/app/?my-secret-admin`
cf. https://github.com/RainLoop/rainloop-webmail/issues/976